backend/inspirehep/submissions/marshmallow/literature.py

```diff
@@ -163,6 +165,7 @@ def build_literature(self, data):
         literature.add_document_type(data["document_type"])
 
         literature.add_arxiv_eprint(data.get("arxiv_id"), data.get("arxiv_categories"))
+        literature.add_preprint_date(data.get("preprint_date"))
```

Comment:
add a test for the ``None`` case

Reply:
all other unit tests expect `test_load_preprint_date`,  are already testing when this is `None`.